common/config/azure-pipelines/ci.yaml

```diff
@@ -26,7 +26,7 @@ schedules:
 jobs:
   - template: jobs/ci-core.yaml
     parameters:
-      name: Node_16
-      nodeVersion: 16.13.0
+      name: Node_18
+      nodeVersion: 18.9.0
```

Comment:
I'd use 18.x, not quite sure why we set this up as an explicit version in the first place...

Reply:
done

Reply:
well isnt 16.13 the first LTS release and thats why?

Reply:
yes, I realized that in a different reply and this was due to us doing 16.x in the regression pipeline.